index.html

```diff
@@ -3,10 +3,34 @@
 slug: Parse + Open Source
 ---
 
+           </td>
+         </tr>
+         <tr class="repoList">
+           <td>
+             <a href=""><h4>Stack Overflow <ins>Parse-Server</ins> tag</h4><p class="repoDescription">
+               For question relating to the hosting and deployment of Parse Server.</p></a>
```

Comment:
I think we need to be careful here based on the feedback from SO - https://meta.stackoverflow.com/questions/381447/rename-tag-parse-com-to-parse-platform/381579#comment679487_381447

Maybe we use something as simple as:

For questions regarding the parse server component.

Reply:
I've change it to "For code level questions related to Parse Server."